golang-ci/example-project/main.go

```diff
@@ -0,0 +1,30 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+)
+
+func helloHandler(w http.ResponseWriter, r *http.Request) {
+	if r.URL.Path != "/hello" {
+		http.Error(w, "404 not found.", http.StatusNotFound)
+		return
+	}
+
+	if r.Method != "GET" {
+		http.Error(w, "Method is not supported.", http.StatusMethodNotAllowed)
+		return
+	}
+
+	fmt.Fprintf(w, "Hello!")
+}
+
+func main() {
+	http.HandleFunc("/hello", helloHandler) // Update this line of code
```

Comment:
We should remove this comment to avoid any possible confusion. I don't want users to read our example projects and think they should be taking anything from them.